nixos/modules/services/misc/matrix-synapse.xml

```diff
@@ -171,17 +171,19 @@ Success!
    option until a better solution for NixOS is in place.
   </para>
  </section>
- <section xml:id="module-services-matrix-riot-web">
-  <title>Riot Web Client</title>
+ <section xml:id="module-services-matrix-element-web">
+  <title>Element (Riot) Web Client</title>
```

Comment:
This is a bit confusing, I would say Element (formerly known as Riot).

Reply:
This does sound clearer, thanks!